src/app/api/search/__tests__/route.test.ts

```
@@ -0,0 +1,230 @@
+            domain: "example.com",
+            description: "A test bookmark",
+            tags: [],
+            collections: [],
+          },
+        ],
+        count: 1,
+        error: null,
+      }),
+      delete: vi.fn().mockReturnThis(),
+      // Make it thenable for await
+      then: vi.fn().mockImplementation((resolve) => {
```

Biome (2.1.2)

[error] Do not add then to an object.



(lint/suspicious/noThenProperty)